From b6e56c1c450f1b236b621777acde5690f0859181 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:02:04 -0400
Subject: [usb] core/hub: Comments shouldnt be C99 // comment style

Message-id: <1410887463-20674-53-git-send-email-dzickus@redhat.com>
Patchwork-id: 95656
O-Subject: [RHEL7 PATCH 052/591] drivers: usb: core: hub.c: Comments shouldnt be C99 // comment style
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 781b2137e3c50e4465908b2c7ffd8f4c48c3461e
Author: Matthias Beyer <mail@beyermatthias.de>
Date:   Thu Oct 10 23:41:29 2013 +0200

    drivers: usb: core: hub.c: Comments shouldnt be C99 // comment style

    Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/hub.c b/drivers/usb/core/hub.c
index 7fcd23e..79e985f 100644
--- a/drivers/usb/core/hub.c
+++ b/drivers/usb/core/hub.c
@@ -2493,7 +2493,7 @@ error_device_descriptor:
  usb_autosuspend_device(usb_dev);
 error_autoresume:
 out_authorized:
- usb_unlock_device(usb_dev); // complements locktree
+ usb_unlock_device(usb_dev); /* complements locktree */
  return result;
 }
 
@@ -3215,7 +3215,7 @@ int usb_port_resume(struct usb_device *udev, pm_message_t msg)
  if (status == 0 && !port_is_suspended(hub, portstatus))
   goto SuspendCleared;
 
- // dev_dbg(hub->intfdev, "resume port %d\n", port1);
+ /* dev_dbg(hub->intfdev, "resume port %d\n", port1); */
 
  set_bit(port1, hub->busy_bits);
 
-- 
1.7.1